perma_web/perma/templates/user_management/includes/created-link-items.html

```diff
@@ -1,8 +1,4 @@
-{% load local_datetime pipeline %}
-
-{% if in_iframe %}
-    {% stylesheet 'base' %}
-{% endif %}
```

Comment:
This is the only use of the variable ```in_iframe```, so perhaps this should be removed too:
perma_web/perma/views/link_management.py:
 ``` 100:         'in_iframe': request.GET.get('iframe'), ```